LICENSE

```diff
@@ -1,21 +1,202 @@
-Copyright (c) 2014 OpenTable, Inc.
```

Comment:
It's best to keep the attributions around somewhere. Folks typically do that in a NOTICE file when it doesn't make sense to include it in the LICENSE file.